data_structures/stack/.gitignore

```diff
@@ -1,5 +1,34 @@
 
-# Created by https://www.gitignore.io/api/python
+# Created by https://www.gitignore.io/api/macos,python,visualstudiocode
```

Comment:
This `.gitignore` is not being used because it is not in the root of your repo. You should update the `.gitignore` in the root of your repo with these changes instead.